lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -52,7 +52,8 @@ sub index {
     my $only_tagged = $self->param('only_tagged') // 0;
     my $show_tags   = $self->param('show_tags')   // $only_tagged;
     my $group_params = $self->every_param('group');
-
+    my $fullscreen   = $self->param('fullscreen');
+    $self->stash('fullscreen', $fullscreen);
```

Comment:
yeah it is not needed anymore, i used a separate template for full screen when i started this.